Source/NavierStokesBase.cpp

```diff
@@ -3068,21 +3069,15 @@ NavierStokesBase::SyncInterp (MultiFab&      CrseSync,
             auto const& finedata    = fdata.array();
             auto const& coarsedata  = cdata.array();
             int scale_coarse = (interpolater == &protected_interp) ? 1 : 0;
-            amrex::ParallelFor(bx, num_comp, [finedata,dt_clev]
+            amrex::ParallelFor(bx, num_comp, [finedata,coarsedata,dt_clev, scale_coarse]
```

Comment:
@dcoveney These changes in SyncInterp and velocity_advection_update shouldn't be in this PR. It suggests to me that your branch was not based off of a up-to-date development branch. Could you try pulling development and merging it in?